site/content/api/Event-API.md

```diff
@@ -76,4 +81,63 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+In the following these events (currently one) are specified in detail.
+
+
+### Device command readiness notification
+
+By sending an event of this type the receiver is notified that a device is ready to receive a command.
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
+is used to send the command to the appropriate protocol adapter instance.
+
+Such an event is typically issued by the device itself. In future releases the protocol adapters may be able to send this
+event on behalf of devices (to support setups with devices that are not capable of sending such an event).
+
+
+***Device command readiness notification Payload***
+
+The following table provides an overview of the properties an event sender needs to set for a *device command readiness notification* event.
+
+| Name           | Mandatory | Location                 | Type      | Description |
+| :------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type* | yes       | *properties*             | *symbol*  | Must be set to *application/vnd.eclipse-hono-dcr-notification+json* |
+
+The body of the event request MUST consist of a single *AMQP Value* section containing a UTF-8 encoded string representation of a single JSON object having the following members:
+
+| Name                | Mandatory | Type       | Default Value | Description |
+| :-----------------  | :-------: | :--------- | :----------   | :---------- |
+| *source*            | *yes*     | *string*   |               | Value can be either the name of a protocol adapter (`hono-mqtt`, `hono-http`, `hono-kura`) or `device`.|
```

Comment:
I have a hard time seeing any good use for this information.

Reply:
I understand the concern, and @ppatierno asked the same question.

So I can eliminate it, but IMHO the sender of the event is of some importance. E.g. if there are mixed setups where a device itself controls fine grained how long it can receive messages, in contrast to a protocol adapter that does this on a more coarse grained level.

But (as I stated in another conversation already), this is not a 100% sharpened use case, so I am willing to delete it now.

Reply:
What difference does it make to the consumer of the event? Will the application apply a *corrective* factor to the ttd value if the event has been sent by the protocol adapter instead of the device?

Reply:
No, and I understand the concerns about the usability of this field (and remove it).
But I imagine that *a device explicitly indicating it's readiness to receive a message* could be treated with maximum importance (the device would not have sent it if not available just now).

Anyway, let's introduce it later (again) if there should be a use for it.